src/app/gigs/chat/emoji-picker.tsx

```diff
@@ -1,23 +1,32 @@
 'use client';
 
+import data from '@emoji-mart/data';
+import Picker from '@emoji-mart/react';
+import { SmileIcon } from 'lucide-react';
+
 import {
   Popover,
   PopoverContent,
   PopoverTrigger,
 } from '@/components/ui/popover';
-import data from '@emoji-mart/data';
-import Picker from '@emoji-mart/react';
-import { SmileIcon } from 'lucide-react';
+import { cn } from '@/lib/utils';
 
 interface EmojiPickerProps {
   onChange: (value: string) => void;
+  /** This affects the styling of the SmileIcon */
+  className?: string;
 }
 
-export const EmojiPicker = ({ onChange }: EmojiPickerProps) => {
+export const EmojiPicker = ({ onChange, className }: EmojiPickerProps) => {
   return (
     <Popover>
       <PopoverTrigger>
-        <SmileIcon className='h-5 w-5 text-muted-foreground hover:text-foreground transition' />
+        <SmileIcon
+          className={cn(
+            'text-muted-foreground hover:text-foreground h-5 w-5 transition',
```

Comment:
```suggestion
            'text-muted-foreground hover:text-foreground size-5 transition',
```

any time you want to set w and h you can use size

Reply:
love this i had no idea 🤯